mep/accounts/models.py

```diff
@@ -230,6 +231,44 @@ def clean(self):
         if self.account and self.person:
             raise ValidationError('Address must only be associated with one of account or person')
 
+    @classmethod
+    def items_to_index(cls):
+        '''Custom logic for finding items to be indexed when indexing in
+        bulk; only include addresses associated with accounts that
+        have documented latitude and longitude.'''
+        return cls.objects.filter(account__isnull=False,
+                                  location__longitude__isnull=False,
+                                  location__latitude__isnull=False)
+
+    def index_data(self):
+        '''data for indexing in Solr'''
+        index_data = super().index_data()
+        # only addresses associated with accounts and
+        # latitude/longitude should be indexed
+        if not all([self.account, self.location.latitude,
+                    self.location.longitude]):
+            del index_data['item_type']
+            return index_data
+
+        index_data.update({
+            'name_s': self.location.name,
+            'street_address_s': self.location.street_address,
+            'city_s': self.location.city,
+            'postal_code_s': self.location.postal_code,
+            'arrondissement_i': self.location.arrondissement(),
+            'latitude_f': str(self.location.latitude),
+            'longitude_f': str(self.location.longitude),
+            'member_slug_ss': [person.slug for person in
+                               self.account.persons.all()],
+
+            # TODO: still needs:
+            # - care of name, if any
+            # - arrondissement number
+            # - associated dates if known
+            # - location id for grouping?
```

Comment:
of these:
- care of name seems worth adding and simple
- arr. number I think you mentioned we can just calculate on the frontend, so we don't need to store it two different ways
- associated dates seems like we won't really have for any them, or at least not anytime soon
- location id we might need if the leaflet marker grouping doesn't work out, but don't think we need right now

Reply:
I already added arrondissement number, just forgot to remove it from this list. No need to duplicate the arrondissement calculation logic.